html.go

```diff
@@ -123,6 +130,10 @@ func HtmlRendererWithParameters(flags int, title string,
 		renderParameters.FootnoteReturnLinkContents = `<sup>[return]</sup>`
 	}
 
+	if renderParameters.SanitizedAnchorNameOverride == nil {
+		renderParameters.SanitizedAnchorNameOverride = markdownutils.CreateGitLabAnchor
```

Comment:
Removed.  My intention was to avoid a nil reference, but testing the code with those 3 lines removed works so I guess it wasn't needed.